core/templates/services/loader.service.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A service to show loading screen.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoaderService {
+  // TODO(#9154): Remove static when migration is complete.
+  /**
+   * The static keyword is used here because this service is used in both
+   * angular and angularjs. Since we are using upgradedServices.ts, where a new
+   * instance is created for angularjs and angular will creates a new instance
+   * for the angular part, we end up having two instances of the service.
+   * In order to keep the variables same, static is used until migration is
+   * complete.
+   */
+  static loadingMessageSubject: Subject<string> = new Subject<string>();
+  constructor() {}
+
+  setLoadingMessage(message: string): void {
```

Comment:
Just a thought, could we have an unset method instead of doing a setLoadingMessage('') to get rid of the message? I think it is easier to understand that way and looks cleaner.

Reply:
Yeah, I think that would be better, will make those changes.

Reply:
Also, please add a unit test for this. In general, it is good practice to include unit tests for any code change to make sure it doesn't regress later on.

Reply:
Done